src/test/java/org/prebid/server/it/ApplicationTest.java

```diff
@@ -120,31 +122,28 @@ public void testOpenrtb2AuctionCoreFunctionality() throws IOException, JSONExcep
     @Test
     public void openrtb2MultiBidAuctionShouldRespondWithBidsFromRubiconAndAppnexus() throws IOException, JSONException {
```

Comment:
Pls rename test, as we don't use Rubicon And Appnexus anymore.
Suggest
openrtb2MultiBidAuctionShouldRespondWithMoreThanOneBid``